script/blog-page.js

```diff
@@ -0,0 +1,65 @@
+window.onload = function() {
+  console.log('loaded');
+}
+
+const mainHeadline = document.createElement('h2');
```

Comment:
all of this code should live in the onload function. It might be working, but you'll want to move it inside that function